website/pages/partials/provisioners/common-config.mdx

```diff
@@ -2,6 +2,8 @@ Parameters common to all provisioners:
 
 - `pause_before` (duration) - Sleep for duration before execution.
 
+- `retry` (int) - Max times the provisioner will retry in case of failure.
```

Comment:
May be we could add: `default to zero (0). Zero means an error will not be retried.`